src/types/manifest.ts

```diff
@@ -11,8 +11,8 @@ export const commandSchema = T.Object({
 
 export const manifestSchema = T.Object({
   name: T.String({ minLength: 1 }),
-  description: T.String({ default: "" }),
-  commands: T.Record(T.String(), commandSchema, { default: {} }),
+  description: T.Optional(T.String({ default: "" })),
+  commands: T.Optional(T.Record(T.String(), commandSchema, { default: {} })),
```

Comment:
Since you have a default, is the Optional needed here?

Reply:
That's what I thought but it turns out that this way of doing it is only correct if you are using `Value.Default` then the TS type will be correct. 
But `createPlugin` accepts manifest as a parameter so I don't use `Value.Default` and if I didn't use `T.Optional`, it would require the user to fill description even though the default is empty string.